kadmin/get.c

```diff
@@ -82,6 +82,8 @@ struct get_entry_data {
     uint32_t mask;
     uint32_t extra_mask;
     struct field_info *chead, **ctail;
+    const char *krb5_config_fname;
+    uint32_t n;
```

Comment:
This is a counter for which principal we're getting when `get` is used to get more than one principal.  We use this in `write_krb5_config()` to append a counter to the given config file name.

Reply:
Add a comment or give this a better name.

Reply:
@vdukhovni doesn't like the `%s-%u` thing, so let's replace this with an `ext_config` that takes a single principal argument and outputs to `stdout`.